### ISSUE TITLE
ImportError: cannot import name 'InsecureRequestWarning'

### ISSUE THREAD

**Issue by sluTTYusr** [thp/urlwatch#159]
I would like to try urlwatch but I get an error after following the installation instructions.
I'm on Jessie with python 3.4.2.

root@123:~# urlwatch
Traceback (most recent call last):
  File "/usr/local/bin/urlwatch", line 48, in <module>
    from urlwatch.command import UrlwatchCommand
  File "/usr/local/lib/python3.4/dist-packages/urlwatch/command.py", line 39, in <module>
    from .jobs import JobBase
  File "/usr/local/lib/python3.4/dist-packages/urlwatch/jobs.py", line 39, in <module>
    from requests.packages.urllib3.exceptions import InsecureRequestWarning
ImportError: cannot import name 'InsecureRequestWarning'

What am I doing wrong?

**Comment by casper** [comment]
Same error here:

```
 from requests.packages.urllib3.exceptions import InsecureRequestWarning
ImportError: No module named 'requests.packages'
```

Installing or upgrading the `requests` package did not help.

**Comment by casper** [comment]
I managed to resolve this problem on Ubuntu 14.04 by uninstalling the system pip (python3-pip) then using `easy-install3` to reinstall the latest pip. After that I was able to upgrade `requests` (which refused to upgrade otherwise). I also needed to manually remove `urllib3` and `chardet` from `/usr/lib/python3/dist-packages`.

In summary: this problem was caused by using the distribution-installed Python 3, which is running a patched (and outdated) pip that will refuse to upgrade any distro-installed Python packages.
